biztown/src/components/AboutSection.js

```diff
@@ -1,241 +1,298 @@
-                        />
-                    </div>
-                </li>
-                <li className="form-group">
-                    <div className="flex-column-container">
-                        <label>NOT including this session, how many times have you participated in JA in the past?</label>
-                        <div className="multiple-choice-answer-container">
-                            <ParticipationComponent
-                                participation={this.props.data.participation}
-                                handleChange={this.props.handleChange}
-                            />
-                        </div>
-                    </div>
-                </li>
-            </ol>
-            </div>
-        );
+  render() {
+    if (!Object.keys(this.props.schoolData).length) {
+      DataManager.getAll("schools").then((data) => {
+        var schoolMap = {};
+        data.results.forEach((school) => {
+          if (!schoolMap[school.district]) {
+            schoolMap[school.district] = [];
+          }
+          schoolMap[school.district].push(school);
+        });
+        this.props.setSchoolData(schoolMap);
+      });
     }
-}
 
-function GradeComponent(props) {
-    var gradeInput = gradeOptions.map(gradeOption => {
-        return (
-            <div
-                className="flex-container"
-                key={gradeOption}
-            >
+    return (
+      <div>
+        <h3>Tell Us About You</h3>
+        <ol>
+          <li className="form-group">
+            <div className="flex-column-container">
+              <label className="form-question">
+                What are the first three letters of your last name?
+              </label>
+              <input
+                className="form-control form-control-text-input"
+                id="name"
+                name="name"
+                type="text"
+                defaultValue={this.props.data.name}
+                maxLength="3"
+                onChange={this.props.handleChange}
+                required
+              />
+            </div>
+          </li>
+          <li className="form-group">
+            <div className="flex-column-container">
+              <label className="form-question">
+                When were you born?
+              </label>
+              <div>
                 <input
-                    className="form-control-radio-btn"
-                    type="radio"
-                    id="grade"
-                    name="grade"
-                    value={gradeOption}
-                    checked={props.grade === gradeOption}
-                    onChange={props.handleChange}
+                  className="form-control"
+                  id="birthDate"
+                  name="birthDate"
+                  type="date"
+                  defaultValue={this.props.data.birthDate}
+                  onChange={this.props.handleChange}
+                  required
```

Comment:
nit: this is also something very low priority, but the form completion alerts display the id as opposed to the actual number or field name.
<img width="263" alt="Screen Shot 2021-03-17 at 1 48 48 AM" src="https://user-images.githubusercontent.com/28936879/111421244-0bd22c00-86c3-11eb-9f47-a5220ced80ed.png">
<img width="273" alt="Screen Shot 2021-03-17 at 1 49 11 AM" src="https://user-images.githubusercontent.com/28936879/111421247-0d9bef80-86c3-11eb-81d5-272f97e24cd6.png">

Reply:
Actually, fixing this should be last priority because it might be slightly annoying to update the handlers to rely on id as opposed to name if you use readable friendly names across the different components.

Reply:
Added to backlog.